tests/test_cli.py

```diff
@@ -980,6 +991,23 @@ def test_run_with_defaults_from_config_file_including_workdir(self, skipper_runn
                                                         interactive=False, net='host', volumes=None,
                                                         workdir='test-workdir')
 
+    @mock.patch('__builtin__.open', create=True)
+    @mock.patch('os.path.exists', autospec=True, return_value=True)
+    @mock.patch('yaml.load', autospec=True, return_value=SKIPPER_CONF_WITH_GIT_REV)
+    @mock.patch('subprocess.check_output', autospec=True, return_value='1234567\n')
```

Comment:
i think you don't check all cases. for exampe, `git_version == True and uncommitted_changes == True`.